styles.css

```diff
@@ -0,0 +1,108 @@
+:root {
+  --icon-font-family: 'Poppins';
+}
+
+body {
+  background-color: black;
+}
+
+.container {
+  display: flex;
+  border: 1px solid blue;
+  width: 375px;
+  height: 100vh;
+  flex-wrap: wrap;
+  flex-direction: column;
+  background-image: url("images/header-shapes mobile@2x.svg");
+  background-repeat: no-repeat;
```

Comment:
- You have applied the background-image in the container class on the main section. This should be preferably applied to the headline class. It depends on how you want to structure your future code viz portfolio section, about-me section, and contact me section. If you intend to put those sections inside the main tag with the container class, you will have a problem.